### DISCUSSION TITLE
本地自动过验证码速度问题

### DISCUSSION THREAD

**Discussion by Colin-Chen630** [mikumifa/biliTickerBuy#127] [提问专区] [answered]
在训练验证码速度界面，在选择了本地过验证码的方式后点击重新生成，那个运行的时间就是当次的验证码用时吗？我每次跑都会在4-6秒，运气不好有几次也有跑了十秒多一点的。是真的需要那么久吗？是的话我可能考虑手动。
![image](https://github.com/mikumifa/biliTickerBuy/assets/87928537/d8dd9d79-cd2c-4423-9b7f-9125e965dd60)

**Comment by Amorter** [comment, marked as answer]
根据网络环境、电脑配置有所不同，10秒多一点是因为第一次验证码验证失败，进行了第二次尝试
可以考虑手动

**Comment by mikumifa** [comment]
如果本地过慢就手动或者rrocr吧，可能和设备有关系
